storybook/src/utils/Gap/Gap.docs.mdx

```diff
@@ -0,0 +1,13 @@
+{/* @license CC0-1.0 */}
+
+import { Controls, Markdown, Meta, Primary } from "@storybook/blocks";
+import * as GapStories from "./Gap.stories.tsx";
+import README from "../../../../packages/css/src/components/gap/README.md?raw";
+
+<Meta of={GapStories} />
+
+<Markdown>{README}</Markdown>
```

Comment:
Relevant for all css util stories: the thing I'd want most from the docs as a developer is a button to copy just the css class, without any of the other stuff. Can we add that?

Reply:
Hah, great minds think alike. I had some local commits on top of this branch for such a thing already :) Needs a bit more work though.

Reply:
Added copyable class names to Gap, Margin, and Visually Hidden, along with visual previews of the tokens.

I chose to use divs and classnames and inline styles for them rather than components. Some of these docs are in a README which isn’t MDX.